gulpfile.js

```diff
@@ -121,7 +122,7 @@ gulp.task('compile-tests', function () {
 
 gulp.task('default', function () {
     runSequence(
-        'watch',
-        $.shell.task('PATH=node_modules/.bin:$PATH electron .')
+        'clean',
+        ['typescript', 'sass', 'react']
```

Comment:
We can use [gulp-if](https://www.npmjs.com/package/gulp-if/) for this, if you too busy with Windows compatibility I can do deeper research and implement functional that you said.